### PR TITLE
Match params of flammable arrow and flaming arrow

### PR DESCRIPTION
**Pull request by ZhilkinSerg**
#### Summary

```SUMMARY: None```

#### Purpose of change

Match params of flammable arrow and flaming arrow

#### Additional context

**Before:**

image

**After:**

image

### CODE REVIEW

**Comment by actual-nh** [comment]
Different materials but the same density?

**Comment by ZhilkinSerg** [comment]
Good catch. Material should be changed too.

**Comment by actual-nh** [comment]
To match the crafting recipe, should be wood - alternatively, change the crafting recipe to be based on a metal arrow.

File: data/json/items/tool/misc.json [hunk 1]
```diff
@@ -195,8 +195,9 @@
     "category": "ammo",
     "name": { "str": "flammable arrow" },
     "description": "This arrow has a rag soaked in a flammable liquid wrapped around the shaft near the head.  You need to light it before shooting.",
-    "weight": "150 g",
-    "volume": "250 ml",
+    "weight": "30 g",
+    "volume": "100 ml",
+    "longest_side": "78 cm",
```
Comment (int-ua) [inline data/json/items/tool/misc.json:200]: It still waits for the change of material, right?

**Pull request merged**